crates/bevy_pbr/src/render/light.rs

```diff
@@ -379,6 +379,14 @@ pub fn extract_lights(
     ) in &directional_lights
     {
         if !view_visibility.get() {
+            commands
+                .get_entity(entity)
+                .expect("Light entity wasn't synced.")
+                .remove::<(
+                    ExtractedDirectionalLight,
+                    RenderCascadesVisibleEntities,
+                    LightViewEntities,
```

Comment:
`LightViewEntities` is added in an observer and not in this system, so it's not clear at the first glance why we remove it here. Maybe we should remove it in an observer too to be consistent? :thinking: